config/default/package.json

```diff
@@ -16,17 +16,17 @@
     "test-unit": "TZ=Africa/Nairobi mocha --recursive --reporter spec test --timeout 20000"
   },
   "devDependencies": {
-    "chai": "^4.2.0",
-    "chai-exclude": "^1.0.8",
+    "chai": "^4.3.8",
+    "chai-exclude": "^2.1.0",
     "chai-like": "^1.1.1",
     "chai-things": "^0.2.0",
-    "cht-conf": "^3.16.2",
-    "cht-conf-test-harness": "^2.4.2",
-    "eslint-plugin-json": "^2.0.1",
-    "mocha": "^5.2.0",
-    "sinon": "^11.1.2"
+    "cht-conf": "^3.21.0",
+    "cht-conf-test-harness": "^2.4.3",
+    "eslint-plugin-json": "^3.1.0",
+    "mocha": "^10.2.0",
+    "sinon": "^12.0.1"
```

Comment:
For the record, I tried bumping to `^16.1.0` to align with the root package.json.  Unfortunately, there is some weirdness between sinon timers being used in the tests and the sinon timers being used in the cht-conf-test-harness.  The bad interaction does not happen for me with Node 18, but does fail with Node 16 (the current CI verison).  

Honestly, does not really seem worth spending too much time to debug at this point since hopefully it is easily resolved when we move to Node 18. However, if you feel it would be best to align versions, I can spend some more time digging into this!